app/server/labml_app/utils/analytics.py

```diff
@@ -70,7 +70,7 @@ def get_meta_data(self, request: Request) -> Dict[str, str]:
         return meta
 
     def track(self, request: Request, event: str, data: Union[NamedTuple, Dict], identifier: str = '') -> None:
-        if isinstance(data, NamedTuple):
+        if isinstance(data, type(NamedTuple)):
```

Comment:
this looks wrong

Reply:
The following error occurs if it's not wrapped with `type()`

``` 
  File "/home/ck/miniconda/lib/python3.9/site-packages/labml_app/utils/analytics.py", line 107, in time_wrapper
    self.track(request, func.__name__, {'time_elapsed': str(total_time)})
  File "/home/ck/miniconda/lib/python3.9/site-packages/labml_app/utils/analytics.py", line 76, in track
    if isinstance(data, NamedTuple):
TypeError: isinstance() arg 2 must be a type or tuple of types
```